### PR TITLE
Implement transactions' add/drop player APIs for 0.4.1

### PR DESCRIPTION
**Pull request by githubsmilo**
issue #6

* add player
```
transactions.add_player(
    league_key,
    team_key,
    player_key,
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```
* drop player
```
transactions.drop_player(
    league_key,
    team_key,
    player_key,
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```
* add and drop players
```
transactions.adddrop_players(
    league_key,
    team_key,
    player_key, // add
    player_key, // drop
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```

### CODE REVIEW

File: collections/transactionsCollection.js [hunk 1]
```diff
@@ -91,3 +91,114 @@ TransactionsCollection.prototype._leagueFetch_callback = function(cb, e, data) {
...
+      url,
+      xmlData,
+      apiCallback
+    );
+};
+
+TransactionsCollection.prototype._drop_player_callback = function(cb, e, data) {
+  if ( e ) return cb(e);
+
+  var meta = data.fantasy_content;
```
Comment (whatadewitt) [inline collections/transactionsCollection.js]: can you provide me with a sample of what this response looks like? I'm wondering if we need anything to act as a "Helper" to clean up the response?

I'm also wondering how I should test these functions as I don't want to actually add/drop players from my teams :D

**Comment by whatadewitt** [comment]
Love this. Now I'm going to need to do the other pieces for the TransactionCollection and TransactionResource :P

File: collections/transactionsCollection.js (hunk 1 above)
Comment (githubsmilo) [inline collections/transactionsCollection.js]: I attached the response of each add/drop APIs. 😸  (I added/dropped my actual player in my team)
* [add_player_API.txt](https://github.com/whatadewitt/yfsapi/files/252271/add_player_API.txt)
* [adddrop_players_API.txt](https://github.com/whatadewitt/yfsapi/files/252272/adddrop_players_API.txt)
* [drop_player_API.txt](https://github.com/whatadewitt/yfsapi/files/252273/drop_player_API.txt)

**Comment by githubsmilo** [comment]
Please share your next schedule. If you need some help, I would like to participate actively.

**Comment by whatadewitt** [comment]
Schedule is very hectic right now with work and a very pregnant wife! I
have some work to catch up on tonight, but I will try to merge this in!
Very excited!

--d

**Comment by whatadewitt** [comment]
@githubsmilo, if you take a look at the other transaction functions, you'll notice that i strip out a lot of the data using the TransactionHelper... Would you mind utilizing the helper to clean up some of the meta so it's easier for users to use?

Thanks!

**Comment by githubsmilo** [comment]
@whatadewitt 
I will take it! :+1:

**Comment by githubsmilo** [comment]
@whatadewitt 
I did strip the result of add/drop player APIs and pushed [a commit](https://github.com/whatadewitt/yfsapi/pull/10/commits/50ea956b3b23715e33da6c3a3e16bc5af057c1ee) to this pull request. Example is below,
* add_player API returns

add_result

* drop_player API returns

drop_result

* adddrop_players API returns

adddrop_result

Please merge this! :smiley:

**Comment by whatadewitt** [comment]
This is great! Thanks!

I'm going to merge this now and bump to v0.5.0 and I will try to spend some time tomorrow night putting together the docs for the sandbox site!  (I'd do it tonight but life is tiresome with a new baby)

**Comment by whatadewitt** [comment]
@githubsmilo any chance you can copy/paste that JSON output so I can use it in the sandbox application?

**Comment by githubsmilo** [comment]
@whatadewitt 
I am sorry that I deleted my JSON output logs. :sob:

**Comment by githubsmilo** [comment]
@whatadewitt 
Some JSON outputs of another examples are here,

* adddrop_players API
```
{"transaction_key":"357.l.161.tr.1503","transaction_id":"1503","type":"add/drop","status":"successful","timestamp":"1464156177","players":[{"player_key":"357.p.9967","player_id":"9967","name":{"full":"Tyler Wilson","first":"Tyler","last":"Wilson","ascii_first":"Tyler","ascii_last":"Wilson"},"editorial_team_abbr":"Bal","display_position":"SP","position_type":"P","transaction_data":{"type":"add","source_type":"freeagents","destination_type":"team","destination_team_key":"357.l.161.t.6","destination_team_name":"Oceans/ Go! Greinke!"}},{"player_key":"357.p.8816","player_id":"8816","name":{"full":"Dillon Gee","first":"Dillon","last":"Gee","ascii_first":"Dillon","ascii_last":"Gee"},"editorial_team_abbr":"KC","display_position":"SP,RP","position_type":"P","transaction_data":{"type":"drop","source_type":"team","source_team_key":"357.l.161.t.6","source_team_name":"Oceans/ Go! Greinke!","destination_type":"waivers"}}]}
```

* drop_player API
```
{"transaction_key":"357.l.161.tr.1504","transaction_id":"1504","type":"drop","status":"successful","timestamp":"1464156358","players":[{"player_key":"357.p.9967","player_id":"9967","name":{"full":"Tyler Wilson","first":"Tyler","last":"Wilson","ascii_first":"Tyler","ascii_last":"Wilson"},"editorial_team_abbr":"Bal","display_position":"SP","position_type":"P","transaction_data":{"type":"drop","source_type":"team","source_team_key":"357.l.161.t.6","source_team_name":"Oceans/ Go! Greinke!","destination_type":"waivers"}}]}
```

* add_player API
```
{"transaction_key":"357.l.161.tr.1505","transaction_id":"1505","type":"add","status":"successful","timestamp":"1464156424","players":[{"player_key":"357.p.9967","player_id":"9967","name":{"full":"Tyler Wilson","first":"Tyler","last":"Wilson","ascii_first":"Tyler","ascii_last":"Wilson"},"editorial_team_abbr":"Bal","display_position":"SP","position_type":"P","transaction_data":{"type":"add","source_type":"freeagents","destination_type":"team","destination_team_key":"357.l.161.t.6","destination_team_name":"Oceans/ Go! Greinke!"}}]}
```

**Pull request merged**